crates/runtime/src/databackend/duckdb.rs

```diff
@@ -20,6 +20,7 @@ use super::{AddDataResult, DataBackend};
 
 #[derive(Debug, Snafu)]
 pub enum Error {
+    #[snafu(display("DuckDBError: {source}"))]
```

Comment:
Is this redundant?

Reply:
No, I discovered that if a display message is specified but the underlying error is not included in the message then it is not added by default. So we should actually start doing this throughout our code.